openhands/runtime/impl/eventstream/eventstream_runtime.py

```diff
@@ -456,7 +457,7 @@ def _wait_until_alive(self):
         ):
             pass
 
-    def close(self, rm_all_containers: bool = True):
+    def close(self):
```

Comment:
I believe we have places (eval?) where we explicitly need to control this...

Reply:
Reverted.